ocean_provider/validation/RBAC.py

```diff
@@ -77,6 +78,16 @@ def get_additional_dids(self):
             for additional_input in additional_inputs
         ]
 
+    def get_data(self):
+        if "data" not in self.request.keys():
+            raise Exception("Data to encrypt is empty.")
+        if not isinstance(self.request["data"], list) and not isinstance(
```

Comment:
please add an empty line before the second `if`, for readability. How come these can be only list and Asset? Is it possible this is a dict or a string json that needs conversion to Asset?